resources/views/user/create.blade.php

```diff
@@ -15,14 +15,19 @@
                             <div class="panel-body">
                                 @include('common.info')
                                 <div class="form-group">
-                                    <label for="user-img" class="control-label">画像</label>
+                                    <label for="image_file" class="control-label">画像
+                                        <span class="badge text-danger">*</span>                                        
+                                    </label>
                                     <div class="input-group">
                                         <span class="input-group-btn">
                                             <label class="btn btn-default border">
-                                                ファイルを選択<input type="file" name="user_img" style="display: none;">
+                                                ファイルを選択<input type="file" name="image_file" style="display: none;" id="image_file" onchange=showFileName()
```

Comment:
```suggestion
                                                ファイルを選択<input type="file" name="image_file" class="d-none" id="image_file" onchange=showFileName()
```
間違いではないので問題ありませんが、bootstrapでも一応実現可能です。

Reply:
ご指摘、ご説明ありがとうございます。
bootstrapを使用したレイアウトへ修正致しました。
ご確認よろしくお願い致します。

resources/views/user/create.blade.php